charts/capsule/templates/deployment.yaml

```diff
@@ -60,8 +60,6 @@ spec:
             secretName: {{ include "capsule.secretTlsName" . }}
       containers:
         - name: manager
-          command:
```

Comment:
May I ask you the reason in removing the command, here?

Reply:
with ko, we no longer build a binary which is called `manager`. We would have to replace it with `/ko-build/capsule`. I thought it's nicer to just remove it.